Gems/EditorPythonBindings/Code/Source/PythonSystemComponent.cpp

```diff
@@ -355,6 +356,16 @@ namespace EditorPythonBindings
                     ;
             }
         }
+
+        if (AZ::BehaviorContext* behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context); behaviorContext)
```

Comment:
nit: this can be simplified to

```
if (auto behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
```

(using `auto` as we alright have the type on the right so no type deduction is taking place and we can avoid repeating the redundant name)